modules/install-sensor-roxctl.adoc

```diff
@@ -18,16 +18,23 @@ ifeval::["{context}" == "install-secured-cluster-cloud-other"]
 :kube:
 endif::[]
 
-To monitor a cluster, you must deploy Sensor.
-You must deploy Sensor into each cluster that you want to monitor.
-The following steps describe adding Sensor by using the {product-title-short} portal.
+To monitor a cluster, you must deploy Sensor. You must deploy Sensor into each cluster that you want to monitor. This installation method is also called the manifest installation method.
+
+To perform an installation by using the manifest installation method, follow _only one_ of these procedures:
```

Comment:
Do we need to supply any information on why you would choose one over the other? 

```suggestion
To perform an installation by using the manifest installation method, follow _only one_ of the following procedures:
```

Reply:
I think in general we are trying to steer people away from using this method at all, and use the Operator or Helm, so I would prefer to leave it as is. (FWIW, I think the second option is used with scripting, but the first requires GUI use.)